docs/src/modules/utils/parseMarkdown.js

```diff
@@ -6,7 +6,7 @@ import prism from 'docs/src/modules/utils/prism';
 
 const headerRegExp = /---[\r\n]([\s\S]*)[\r\n]---/;
 const titleRegExp = /# (.*)[\r\n]/;
-const descriptionRegExp = /<p class="description">(.*)<\/p>[\r\n]/;
+const descriptionRegExp = /<p class="description">(.*)<\/p>/s;
```

Comment:
The trailing `[\r\n]` seems to restrictive to me. Removing it did not include more `<p class="descriptions" />`.

Reply:
Only newline + blankline would have a formatting impact.